connector/protobuf/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I am planning to work on Python support. @mposdev21 @SandishKumarHN let me know if you have already changes for it, otherwise, I can get started.

Reply:
@rangadi we already have the changes ready. should we create a separate PR now?

Reply:
Yes, please go ahead with the PR if you can. We can start reviewing it even as this PR is being merged.